README.md

```diff
@@ -247,7 +248,7 @@ Python plugins developers must ensure their plugin to work with all Python versi
 [summary]: https://github.com/lightningd/plugins/tree/master/archived/summary
 [trustedcoin]: https://github.com/fiatjaf/trustedcoin
 [ts-api]: https://github.com/runcitadel/c-lightning.ts
-[watchtower-client]: https://github.com/talaia-labs/rust-teos/tree/master/watchtower-plugin
+[rust-teos]: https://github.com/talaia-labs/rust-teos/tree/master/watchtower-plugin
```

Comment:
don't rename this, `watchtower-client` is the name of the plugin

Reply:
This also breaks the link and looks super weird
![Screenshot from 2024-02-13 09-46-42](https://github.com/lightningd/plugins/assets/7445670/4674b8d0-93b4-43be-bbcd-fcfd0622c238)

Just revert this change